storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1327,6 +1344,8 @@ bool Redo_Log_Data_Manager::copy_once(bool is_last, bool *finished) {
       if (stop_lsn == 0 || archive_reader.get_contiguous_lsn() >= stop_lsn) {
         return (true);
       }
+    } else {
+      ib::warn() << "Failed to seek Archive log file from LSN " << start_lsn;
```

Comment:
```suggestion
      ib::warn() << "Failed to seek Archive log file from LSN " << start_lsn << ". Will fallback to live redo log";
```

Reply:
fixed

Reply:
When we fallback, we might not have the desired LSN with redo file, do we account for this?

Reply:
It relies on log_scanned_LSN and if it fails to find the file for that LSN. It will reopen the redo file.